java/src/main/java/com/google/cloud/dataproc/templates/gcs/README.md

```diff
@@ -18,6 +18,15 @@ bin/start.sh \
 --templateProperty gcs.bigquery.temp.bucket.name=<bigquery temp bucket name>
 ```
 
+There are two optional properties as well with "GCS to BigQuery" Template. Please find below the details :-
+
+```
+--templateProperty gcs.bigquery.temp.table='dataset' 
```

Comment:
is 'dataset' the best name for the table?

Reply:
Done. Have changed it to "table_name" instead of "dataset"